### PR TITLE
Added one more rule for response validation in model validation.

### PR DESCRIPTION
**Pull request by raych1**
Fail the validation when there's schema defined for response body in spec but the example
response doesn't provide the body in response.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Changelog
+### 07/02/2019 0.18.5
```
Comment (nschonni) [inline ChangeLog.md]: 0.18.5 was published on June 12th d54eaf3cfd2a89bfb5da9878d4b129add281996d
```suggestion
### 06/12/2019 0.18.5
```

Comment (raych1) [reply ChangeLog.md]: Good catch, thanks!

File: ChangeLog.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # Changelog
+### 07/02/2019 0.18.6
+- Fail the validation when there's schema defined for response body in spec but the example response doesn't provide the body.
+
+### 06/12/2019 0.18.5
```
Comment (sergey-shandar) [inline ChangeLog.md]: It should be
```
- Adding support for JSON path & schema path in errors
```

File: lib/validators/modelValidator.ts [hunk 3]
```diff
@@ -558,20 +558,37 @@ export class ModelValidator extends SpecValidator<SpecValidationResult> {
...
+
+      // Fail when example provides the response body but the swagger spec doesn't define the schema for the response.
+      if (exampleResponseBody !== undefined && !response.schema) {
         const msg =
           `Response statusCode "${exampleResponseStatusCode}" for operation ` +
           `"${operation.operationId}" has response body provided in the example, ` +
           `however the response does not have a "schema" defined in the swagger spec.`
         const e = this.constructErrorObject<Sway.ValidationEntry>({
-          code: C.ErrorCodes.ResponseSchemaNotInSpec,
+          code: C.ErrorCodes.ExtraResponseBodyInExample,
```
Comment (amarzavery) [inline lib/validators/modelValidator.ts]: `C.ErrorCodes.ResponseSchemaNotInSpec` is there to indicate the same thing `that a response is provided in example for which the schema is not defined in the swagger spec`. So please replace the code `ExtraResponseBodyInExample` with `ResponseSchemaNotInSpec`.

File: lib/util/constants.ts [hunk 4]
```diff
@@ -96,6 +96,14 @@ export const ErrorCodes = {
   DoubleForwardSlashesInUrl: {
     name: "DOUBLE_FORWARD_SLASHES_IN_URL",
     id: "OAV129"
+  },
+  ResponseBodyNotInExample: {
+    name: "RESPONSE_BODY_NOT_IN_EXAMPLE",
+    id: "OAV130"
+  },
+  ExtraResponseBodyInExample: {
```
Comment (amarzavery) [inline lib/util/constants.ts]: This is not required. Please remove this.

File: ChangeLog.md (hunk 2 above)
Comment (raych1) [reply ChangeLog.md]: Updated.

File: lib/util/constants.ts (hunk 4 above)
Comment (raych1) [reply lib/util/constants.ts]: Removed and use the original one.

**Comment by raych1** [comment]
@amarzavery , the fix was submitted, please review.

**Pull request merged**